### PR TITLE
Implement class progress management

### LINKED ISSUE #19: As a form teacher, I want to maintain a list of students who have completed their homework
so that I can track who has not submitted their work.

### PR DESCRIPTION
**Pull request by tingalinga**
Added Homework class

### CODE REVIEW

**Comment by Yuanyu97** [comment]
Good job starting on the implementation of the homework feature. I noticed you used LocalDateTime to parse your date instead of just using a string, good job on that too. For the next step, you could maybe use another data type to keep track of the progress. As of now, you are using boolean to keep track of students' submission of homework. In the next version, maybe you could use an enum class to keep track of the grades of the homework achieved by the student. Also, maybe you could have an abstract Academic class that keeps track of both homework and examination progress of students. Overall, a job well done and keep up the good work!

**Review by Yuanyu97 — COMMENTED** [review]
Good job starting on the implementation of the homework feature. I noticed you used LocalDateTime to parse your date instead of just using a string, good job on that too. For the next step, you could maybe use another data type to keep track of the progress. As of now, you are using boolean to keep track of students' submission of homework. In the next version, maybe you could use an enum class to keep track of the grades of the homework achieved by the student. Also, maybe you could have an abstract Academic class that keeps track of both homework and examination progress of students. Overall, a job well done and keep up the good work!

**Comment by tingalinga** [comment]
Thank you for your insightful feedback Circle fish! I was thinking of creating a Submission class to replace the boolean attribute so that I can also keep track of whether the submission was late or on time. Will definitely think about implementing abstract class to allow for different types of assessments in the future!

File: src/main/java/seedu/address/ui/UiManager.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
     public static final String ALERT_DIALOG_PANE_FIELD_ID = "alertDialogPane";
 
     private static final Logger logger = LogsCenter.getLogger(UiManager.class);
-    private static final String ICON_APPLICATION = "/images/address_book_32.png";
+    private static final String ICON_APPLICATION = "/images/teapot_icon.png";
```
Comment (TheElectricBlender) [inline src/main/java/seedu/address/ui/UiManager.java:23]: Was looking forward to this implementation. Cant wait to see how its gonna look!!

**Comment by TheElectricBlender** [comment]
Good start on the current implementation of the homework tracker! @Yuanyu97 hopefully no merge conflicts when merging.
If not, looks good to merge!

**Pull request merged**